Sources/Sentry/SentrySessionTracker.m

```diff
@@ -72,4 +80,9 @@ - (void)willResignActive {
     self.lastInForeground = [NSDate date];
 }
 
+- (void)willTerminate {
+    NSDate *from = nil == self.lastInForeground ? [NSDate date] : self.lastInForeground;
```

Comment:
`l`: Nice one-liner ^^. The naming for `from` is a little bit confusing for me. I don't really get it.

Reply:
Fair. It was the name taken before where there's some `from` `to` calculation.
I've renamed it here: https://github.com/getsentry/sentry-cocoa/pull/506